src/main/resources/swagger-current.yml

```diff
@@ -3,7 +3,7 @@ info:
   title: ONS Contact Centre API
   version: "5.11.18-oas3"
```

Comment:
Let's reset the version to 1.0.0 and the title to "ONS SDC Contact Centre API"

Reply:
Done. Also removed the (old) history, since that would just be confusing.